litmus-portal/backend/auth/go.sum

```diff
@@ -13,19 +13,25 @@ github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSs
 github.com/dgrijalva/jwt-go v3.2.0+incompatible h1:7qlOGliEKZXTDg6OTjfoBKDXWrumCAMpl/TFQ4/5kLM=
```

Comment:
Remove this change

Reply:
Hello @rajdas98  thank you for your review, Yes removed that!